beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -362,6 +416,11 @@ func (vs *Server) blobSidecarsFromUnblindedBlock(block interfaces.SignedBeaconBl
 	return BuildBlobSidecars(block, rawBlobs, proofs)
 }
 
+// broadcastReceiveChunkedBlock broadcasts a chunked block and handles its reception.
+func (vs *Server) broadcastReceiveChunkedBlock(ctx context.Context, req *ethpb.ChunkedBeaconBlock, root [32]byte) error {
+	return nil
```

Comment:
This needs to be implemented like mentioned on gossip: instead of calling broadcast for  a single message, each peer needs to get a different chunk.